tests/foreman/virtwho/ui/test_hyperv_sca.py

```diff
@@ -0,0 +1,120 @@
+"""Test class for Virtwho Configure UI
+
+:Requirement: Virt-whoConfigurePlugin
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Virt-whoConfigurePlugin
+
+:Team: Phoenix
+
+:TestType: Functional
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+
+from robottelo.config import settings
+from robottelo.utils.virtwho import deploy_configure_by_command
+from robottelo.utils.virtwho import deploy_configure_by_script
+from robottelo.utils.virtwho import get_configure_command
+from robottelo.utils.virtwho import get_configure_file
+from robottelo.utils.virtwho import get_configure_id
+from robottelo.utils.virtwho import get_configure_option
+
+
+@pytest.fixture()
+def form_data():
+    form = {
+        'debug': True,
+        'interval': 'Every hour',
+        'hypervisor_id': 'hostname',
```

Comment:
Just confirming if we can have a hostname as hypervisior_id?

Reply:
@adarshdubey-star Yes, hypervisor_id can have hostname, uuid, hwuuid these three values